persistence/sql/persister_identity.go

```diff
@@ -139,7 +144,13 @@ func (p *Persister) CreateIdentity(ctx context.Context, i *identity.Identity) er
 
 func (p *Persister) ListIdentities(ctx context.Context, limit, offset int) ([]identity.Identity, error) {
 	is := make([]identity.Identity, 0)
-	return is, sqlcon.HandleError(p.c.RawQuery("SELECT * FROM identities LIMIT ? OFFSET ?", limit, offset).All(&is))
+	err := sqlcon.HandleError(p.c.RawQuery("SELECT * FROM identities LIMIT ? OFFSET ?", limit, offset).All(&is))
+
+	for i := range is {
+		p.injectTraitsUrl(&is[i])
```

Comment:
This definitely needs a test. Pointers behave unexpectedly in loops, I think this is fine as it is but make sure to test it!